Makefile

```diff
@@ -3,13 +3,15 @@ build: check funktion-operator
 REPO = fabric8io/funktion-operator
 TAG = latest
 GO := GO15VENDOREXPERIMENT=1 go
+BUILD_DIR ?= ./out
+NAME = funktion-operator
 
 funktion-operator: $(shell find . -type f -name '*.go')
 	go build -o funktion github.com/fabric8io/funktion-operator/cmd/operator
 
 funktion-linux-static: $(shell find . -type f -name '*.go')
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
-		go build -o funktion-linux-static \
+		go build -o ./out/funktion-operator-linux-amd64 \
```

Comment:
@jstrachan this was needed so that the existing `make image` still works.  I dont think it's an issue but I just wanted to point it out.